CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+# Contributors
+
+[Gregory McQuillan](http://github.com/hk0i)
```

Comment:
sure. should we all go under contributors or should we have a separate `AUTHORS` file for original authors? Either way I don't mind the extra company